Documentation/UserGuide.md

```diff
@@ -1,27 +1,86 @@
-# User Guid for p, the genral project manager
-Many languages have an own project manager.
+# User Guide for p, the general project manager
+Many languages have their own project manager.
 Some examples are cargo for rust, elm for elm,
-lein for clojure, you name it.
+lein for clojure,
+you name it.
 
-But did you ever try to use cargo as you use elm
-or lein as you use elm?
-If you did, I'm sorry for your failure.
+But did you ever try to use cargo as
+you use elm?
+You can't use one project manager as
+you use another one.
 
-One project manager for one language isn't
-interchangeable with the project manager
-of another language.
 This adds extra challenges to
 learning a new language.
 An unnecessary challenge, I think.
 
-I think the problem is now explained
-sufficently and you know what problem
-p tries to solve.
+## Install
 
-With p you will be able to create a project
-for any (supported) language with this command:
+p works on:
+
+| OS    | Version of Python | Other requirements |
```

Comment:
Does p work with anything else? 
I couldn't say, this is just my setup of p and I thought it worked.

Reply:
I've only tested on macOS so that works.

Reply:
And Python 3.7.3, or lower?
And anything else installed to run it?

Reply:
3.6 should suffice. I use f-strings.